obj/BaseObj.gd

```diff
@@ -580,8 +580,8 @@ func set_facing(facing: int):
 	if facing != 1 and facing != -1:
 		facing = 1
 	chara.set_facing(facing)
-	if is_instance_valid(flip):
-		flip.scale.x = -1 if facing < 0 else 1
+	if is_instance_valid($Flip):
```

Comment:
I realized in hindsight you could ``get_node_or_null("Flip")`` here, but considering BaseObj.tscn *always* has a Flip node, I didn't bother to figure out how to edit the commit